src/index.tsx

```diff
@@ -1,5 +1,6 @@
-import './assets/styles/main.scss';
+// import './assets/styles/main.scss';
```

Comment:
Why do you have to comment this out?

Reply:
It didn't seem to do anything and I considered global styles should come from hds-core.

Reply:
I uncommented that file and decided to move font face definitions there. However, in the long run I'd like to get rid of the `assets/` folder entirely.